do_more_chapter_6/Readme.md

```diff
@@ -4,12 +4,14 @@
 ## Do More
 
 ### On Unix, try the ls -lR command while you're in temp.
+  `ls -lr ` : list directories with permissions, links , owner, group , size , time and name
 
-    Please use markdown formatting listing the command(s) you ran to accomplish this
 
 ### Use cd to get to other directories on your computer then use ls to see what's in them.
 
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+   `ls -A`
```

Comment:
Please provided the `cd` command you used to get to "other directories", then use `ls` to see what's in them and provide the output.